uint8 state_ocu
float32 voltage_24v_channel
float32 voltage_12v_channel
float32 voltage_7_4v_channel
float32 current_drivetrain_left
float32 current_drivetrain_right
float32 current_robot_arm
float32 current_24v_aux
float32 current_main_comp
float32 current_payload
float32 current_12v_aux_1
float32 current_12v_aux_2
float32 current_steering_front_left
float32 current_steering_front_right
float32 current_steering_rear_left
float32 current_steering_rear_right
float32 temperature_ocu_board_24v
float32 temperature_ocu_board_12v
float32 temperature_ocu_board_7_4v
float32 cell_voltage_1
float32 cell_voltage_2
float32 cell_voltage_3
float32 cell_voltage_4
float32 cell_voltage_5
float32 cell_voltage_6
float32 voltage_science_payload
float32 current_science_payload
float32 temperature_science_payload
float32 apd_bias_voltage
float32 apd_temperature
uint32 apd_rlu
float32 position_drum
float32 weight
